lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.spec.ts

```diff
@@ -890,13 +891,17 @@ describe('StartProcessCloudComponent', () => {
             expect(title).toBeFalsy();
         });
 
-        it('should show title', () => {
+        it('should show title', async () => {
```

Comment:
Since we are not `await` here for anything, the `async ` is not needed.

Reply:
fixed in new commit